internal/config/config.go

```diff
@@ -81,7 +83,7 @@ func InitConfig() {
 		viper.SetDefault("DBHost", "localhost")
 		viper.SetDefault("DBPort", "15432")
 	}
-
+	viper.SetDefault("API_PORT", ":8088")
```

Comment:
We should not have colon `:` in the `API_PORT` env var.

Reply:
Changes pushed